cmake/scripts/common/Macros.cmake

```diff
@@ -777,7 +777,15 @@ endmacro()
 # find all folders containing addon.xml.in and used to define
 # ADDON_XML_OUTPUTS, ADDON_XML_DEPENDS and ADDON_INSTALL_DATA
 macro(find_addon_xml_in_files)
+  set(filter ${ARGN})
+
   file(GLOB ADDON_XML_IN_FILE ${CMAKE_SOURCE_DIR}/addons/*/addon.xml.in)
+  if(filter)
+    if(VERBOSE)
+      message(STATUS "find_addon_xml_in_files: filtering ${filter}")
+    endif()
+    list(FILTER ADDON_XML_IN_FILE EXCLUDE REGEX "${filter}")
```

Comment:
isn't correct as the filter **list** could have more then one item

Reply:
I made it a single regex now, no need for a list and another loop